packages/form-builder/src/App.tsx

```diff
@@ -33,6 +33,7 @@ const initialState = {
         ),
         primaryColor: '#69b86b',
         secondaryColor: '#f49420',
+        pageSlug: 'new-form',
```

Comment:
This is really just a placeholder. The default value is generated by the server for uniqueness when the form is created (which is before the builder loads for the first time).